client/modules/users/userReducer.js

```diff
@@ -34,6 +34,24 @@ export const editUser = user => ({
   }
 })
 
+export const demoteUser = userId => ({
+  [CALL_API]: {
+    endpoint: `admin/users/${userId}/demote`,
+    method: 'PUT',
+    schema: userSchema,
+    types: [actions.SAVE_REQUEST, actions.SAVE_SUCCESS, actions.SAVE_FAILURE]
+  }
+})
+
+export const promoteUser = userId => ({
+  [CALL_API]: {
+    endpoint: `admin/users/${userId}/promote`,
```

Comment:
I don't think this endpoint exists yet in the server routes

Reply:
It was there before, but when I merged the behavior into `profile.update()` I didn't fully cleanup the old routes.